sklearn/datasets/_twenty_newsgroups.py

```diff
@@ -391,23 +394,38 @@ def fetch_20newsgroups_vectorized(*, subset="train", remove=(), data_home=None,
 
         .. versionadded:: 0.22
 
+    as_frame : bool, default=False
+        If True, the data is a pandas DataFrame including columns with
+        appropriate dtypes (numeric, string or categorical). The target is
+        a pandas DataFrame or Series depending on the number of target_columns.
+
```

Comment:
A second thought. we could add a little note here to mention that the dataframe will be sparse and that it will require pandas 0.25+